src/classes/modules/java.base/java/lang/String.java

```diff
@@ -488,4 +488,18 @@ private static String generateStringByConcatenatingArgs(Object[] args) {
     }
     return builder.toString();
   }
+
+
+	/**
+	 * StringIndexOutOfBoundsException  if {@code index} is
+	 * negative or greater than or equal to {@code length}.
+	 *
+	 * Added to support SAXParserTest
+	 */
+  static void checkIndex(int index, int length) {
+	  if (index < 0 || index >= length) {
+	    throw new StringIndexOutOfBoundsException("index " + index + ",length " + length);
+		}
```

Comment:
Please fix the indentation here.

Reply:
The indentation has now been fixed, Cyrille.
I'm sorry I did not catch it sooner, this looked good in the IDE but when I used vi to look at the code, the spacing issue did exist.
Thanks!